oggm/workflow.py

```diff
@@ -147,7 +147,7 @@ def execute_entity_task(task, gdirs, **kwargs):
         if ogmpi.OGGM_MPI_COMM is not None:
             return ogmpi.mpi_master_spin_tasks(pc, gdirs)
 
-    if cfg.PARAMS['use_multiprocessing']:
+    if cfg.PARAMS['use_multiprocessing'] and cfg.PARAMS['mp_processes'] > 1:
```

Comment:
The issue here is that there are two other ways it could be 1.
Once if the system actually only has a single CPU, and one if the slurm env var is 1.

Reply:
> Once if the system actually only has a single CPU, and one if the slurm env var is 1.

But in this cases, we also would like to turn off multiprocessing altogether right? I thought this was the point of your comment on slack yesterday

Reply:
Yes, that's what I'm saying.
If the CPU count is set to 1, there is no reason to even bother with multiprocessing.
And mp_processes is not the only way it can be set to 1.